source/plugin/Makefile

```diff
@@ -169,6 +170,11 @@ $(BINDIR)/CarlaPatchbayFX$(LIB_EXT): $(OBJDIR)/carla-vst.cpp.patchbay-fx.o $(LIB
 	@echo "Linking CarlaPatchbayFX$(LIB_EXT)"
 	@$(CXX) $< $(LIBS_START) $(LIBS) $(LIBS_END) $(SHARED) $(LINK_FLAGS) -o $@
 
+$(BINDIR)/CarlaVstShell$(LIB_EXT): $(OBJDIR)/carla-vst.cpp.vst-shell.o $(LIBS)
```

Comment:
small nitckpick, I would name this object carla-vst.cpp.shell.o, taking out the -vst suffix.
It is already implied it is a vst :)

Reply:
OK, I'll do that.